packages/jsii-config/lib/index.ts

```diff
@@ -0,0 +1,66 @@
+import * as yargs from 'yargs';
+import { readFile, writeFile } from 'fs';
+import getAnswers from './prompt';
+
+const readFilePromise = (path: string): Promise<Buffer> => {
+  return new Promise((resolve, reject) => {
+    readFile(path, (err, data) => {
+      if (err) {
+        return reject(err);
+      }
+      return resolve(data);
+    });
+  });
+};
+
+const writeFilePromise = (path: string, config: string): Promise<void> => {
+  return new Promise((resolve, reject) => {
+    writeFile(path, config, err => {
+      if (err) {
+        return reject(err);
+      }
+      return resolve();
+    });
+  });
+};
+
+/*
+ * Read package.json and prompt user for new or revised jsii config values.
+ */
+export default async function main() {
```

Comment:
I did this to keep as much surface area as possible testable with jest without reaching into /bin. I moved main() to the bin directory and slightly changed to the signature for the main /lib export to avoid losing coverage.